priv/templates/boot.eex

```diff
@@ -613,11 +613,11 @@ case "$1" in
             mv "$REL_DIR/vm.args.bak" "$REL_DIR/vm.args"
         fi
         # Then, backup the packaged configs
-        cp "$REL_DIR/sys.config" "$REL_DIR/sys.config.bak"
-        cp "$REL_DIR/vm.args" "$REL_DIR/vm.args.bak"
+        cp -afv "$REL_DIR/sys.config" "$REL_DIR/sys.config.bak"
```

Comment:
I think what we really want here is `cp --preserve=links ...`, but I could see an argument for `-a` (i.e. `--preserve=all`). I'm not a fan of `-f` because it hides unexpected failures, which we definitely don't want in the case of safely backing up an original file. I also would like to remove the `-v`, as we already have the debugging mode available to show us what's happening in detail if troubleshooting, and I'm of the opinion that verbose output is only useful in exceptional cases, not common ones.

If you could change this from `-afv` to `-a`, I'll get this merged :)